templates/zerver/help/searching-for-messages.md

```diff
@@ -0,0 +1,112 @@
+# Searching for messages
+
+In Zulip, you can find specific messages by using the search bar (highlighted in
+red in the image below) at the top of your screen. This feature allows you to
+narrow your view to show specific messages using search constraints called
+**operators**.
+
+![Search bar](/static/images/help/search-bar.png)
+
+Operators are short phrases in the format `operator:operand` where `operand`
+represents the criteria that fits the specified operator. Operators allow you to
+instantly find messages that meet the specified criteria, such as messages that
+belong to a specific stream or topic.
+
+It is important to note that spaces in `operand` must be replaced with `+`. In
+addition, multiple search operators can be used in a single query by separating
+each operator with spaces in your search bar. Most importantly, you can search
+an individual stream for results from before you joined, but search results will
+only include messages from after you first joined Zulip if you search the entire
+realm.
+
+## Features and limitations
+
+* Search hits include morphological variants by default; for example, if you
+search for messages that contain "walking", you’ll also find messages that
+contain "walk", "walks", "walked", etc.
+* Searches are case-insensitive.
+* Searching by date currently is not possible.
+* Some common words might be handled as “stop words” — the search tool will
```

Comment:
Maybe give an example or link to the Wikipedia article explaining what are stop words?